Documentation/hacking.md

```diff
@@ -207,7 +207,47 @@ Now, GOTO [saving](#saving-the-set-of-dependencies)
 
 #### Removing an existing dependency
 
-This is the simplest case of all: simply remove all references to a dependency from the source files.
+This is almost the simplest case of all: simply remove all references to a dependency from the source files.
+
+Now, GOTO [saving](#saving-the-set-of-dependencies)
+
+#### Add a new application
```

Comment:
Could you add a little more context about what an "application" is (i.e. a go main package), and perhaps why one might want to vendor it